src/commands/convert/asff2hdf.ts

```diff
@@ -4,52 +4,161 @@ import {ASFFResults as Mapper} from '@mitre/hdf-converters'
 import {checkSuffix} from '../../utils/global'
 import _ from 'lodash'
 import path from 'path'
+import AWS from 'aws-sdk'
+import https from 'https'
+import {AwsSecurityFindingFilters} from 'aws-sdk/clients/securityhub'
+import {createWinstonLogger} from '../../utils/logging'
 
 export default class ASFF2HDF extends Command {
-  static usage =
```

Comment:
remind me to ask @charleshu-8 or @Rlin232 to look at that usage statement website and rework all of our usage statements.  for now it's fine to remove, but we should add it back in soon.